bigtable-client-core-parent/bigtable-client-core/src/main/java/com/google/cloud/bigtable/grpc/scanner/RowMerger.java

```diff
@@ -377,6 +377,13 @@ public RowMerger(StreamObserver<FlatRow> observer) {
     this.observer = observer;
```

Comment:
Might better to call reset() in the constructor and remove the variable initializers to avoid duplicating initialization